Library/Homebrew/language/python.rb

```diff
@@ -304,7 +304,7 @@ def pip_install_and_link(targets, link_manpages: false)
         def do_install(targets)
           targets = Array(targets)
           @formula.system @venv_root/"bin/pip", "install",
-                          "-v", "--no-deps", "--no-binary", ":all:",
+                          "-v", "--no-deps", "--use-feature=no-binary-enable-wheel-cache",
```

Comment:
Nice. What versions are supported by the `no-binary-enable-wheel-cache` feature?

Reply:
According to this (https://github.com/pypa/pip/pull/11454), 22.3.

This will become the default behaviour in the future, but we can opt into it now.

Reply:
Sounds good. We'll need to wait until `python@3.10` is on 22.3+, though. That's in Homebrew/homebrew-core#117507.

Reply:
I wonder - is it possible to bump just `pip`? That should be fairly quick and painless compared to the larger change in https://github.com/Homebrew/homebrew-core/pull/117507.

If this change works as expected, it might also make some of the dependent builds faster I think?

I can make a MR for that if you'd like

Reply:
I doubt it'll go much faster that way, TBH. It will need to wait for a long build slot anyway.